pkg/bootstrapper/service.go

```diff
@@ -90,6 +117,28 @@ func (service *kubeletService) controlService(cmd svc.Cmd, desiredState svc.Stat
 	return nil
 }
 
+// controlDependentService sends a control signal to the kubelet dependent service
+func controlDependentService(serviceObj *mgr.Service, cmd svc.Cmd, desiredState svc.State) error {
+	status, err := serviceObj.Control(cmd)
+	if err != nil {
+		return err
+	}
+	// Most of the rest of the function borrowed from the package (golang.org/x/sys/windows/svc/mgr) example
+	// Arbitrary wait time
```

Comment:
Is it? There is definitely a time you are waiting here. Only `serviceWaitTime` is arbitrary.